src/sass/_direction_specific_input.scss

```diff
@@ -0,0 +1,11 @@
+.direction-specific-input {
+  align-items: center;
+  display: grid;
+  gap: $sp-medium;
+  grid-template-columns: fit-content(50%) 1fr;
+  padding-bottom: $spv--large;
+
+  .direction-specific-key {
+    overflow: hidden;
+  }
```

Comment:
```suggestion
```
I don't think we need this?
Maybe we can remove both of the `direction-specific-key` and `direction-specific-value` classes completely?